libbeat/common/uuid.go

```diff
@@ -0,0 +1,22 @@
+package common
+
+import (
+	"crypto/rand"
+	"fmt"
+)
+
+// GenerateUUID is used to generate a random UUID
+// This was copied from https://github.com/hashicorp/uuid/blob/master/uuid.go
```

Comment:
@urso Should we go with an external lib with a better license or our own implementation? As a first step I would suggest to pick an other external implementation and bring it in house if needed later. Agree?